docsearch.config.json

```diff
@@ -7,6 +7,7 @@
         "version": [
           "latest",
           "stable",
+          "v1.8",
           "v1.7",
           "v1.6",
           "v1.5"
```

Comment:
@genbit any reason to index v1.5? v1.5 isn't using the newest theme anyway.

Reply:
no